Bio/GenBank/Scanner.py

```diff
@@ -352,7 +356,7 @@ def parse_footer(self):
             if line == "//":
                 break
         self.line = line
-        return ([], "")  # Dummy values!
+        return [[], ""]  # Dummy values!
```

Comment:
In fact according to CodeCov, this line isn't covered by any of our tests at all :(